src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -42,7 +42,7 @@ internal SdkDirectoryWorkloadManifestProvider(string sdkRootPath, string sdkVers
             _sdkRootPath = sdkRootPath;
             _sdkVersionBand = new SdkFeatureBand(sdkVersion);
 
-            var knownManifestIdsFilePath = Path.Combine(_sdkRootPath, "sdk", sdkVersion, "IncludedWorkloadManifests.txt");
+            var knownManifestIdsFilePath = Path.Combine(_sdkRootPath, "sdk", sdkVersion, "KnownWorkloadManifests.txt");
```

Comment:
I think for the time being we should look for both files, so that we don't have to make the change in dotnet/installer exactly when the SDK with this change flows in.

Reply:
Updated but tried to keep the logic as similar as possible. You could have issues if one txt file has some workloads and the other has different ones AND the order they load in matters. I can't imagine when we'd end up in that state. I do try to avoid duplicates at least.